### PR TITLE
chore: remove unused eslint disable comments

### PR DESCRIPTION
**Pull request by imranbarbhuiya**
🧹

### CODE REVIEW

File: packages/discord.js/src/managers/GuildManager.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ class GuildManager extends CachedManager {
     return super.resolveId(guild);
   }
 
-  /* eslint-disable max-len */
+
```
Comment (kyranet) [inline packages/discord.js/src/managers/GuildManager.js:140]:
```suggestion
```

File: packages/discord.js/src/structures/Guild.js [hunk 2]
```diff
@@ -727,7 +727,7 @@ class Guild extends AnonymousGuild {
     return new GuildAuditLogs(this, data);
   }
 
-  /* eslint-disable max-len */
+
```
Comment (kyranet) [inline packages/discord.js/src/structures/Guild.js:730]:
```suggestion
```

File: packages/discord.js/src/structures/MessagePayload.js [hunk 3]
```diff
@@ -128,7 +128,7 @@ class MessagePayload {
     let nonce;
     if (typeof this.options.nonce !== 'undefined') {
       nonce = this.options.nonce;
-      // eslint-disable-next-line max-len
+
```
Comment (kyranet) [inline packages/discord.js/src/structures/MessagePayload.js:131]:
```suggestion
```

File: packages/discord.js/src/client/websocket/WebSocketShard.js [hunk 4]
```diff
@@ -591,7 +591,7 @@ class WebSocketShard extends EventEmitter {
       }
 
       this.debug(
-        // eslint-disable-next-line max-len
+
```
Comment (kyranet) [inline packages/discord.js/src/client/websocket/WebSocketShard.js:594]:
```suggestion
```

**Review by kyranet — DISMISSED** [review]

**Comment by imranbarbhuiya** [comment]
> Why does this modify the lock file? Did you not install with `--immutable`?

fixed

**Review by iCrawl — APPROVED** [review]

**Pull request merged**